fxa-client/src/lib.rs

```diff
@@ -653,7 +653,14 @@ mod tests {
         let url = fxa.begin_pairing_flow(&PAIRING_URL, &["https://identity.mozilla.com/apps/oldsync"]);
 
         assert!(url.is_err());
-        assert_eq!(format!("{:?}", url), "Err(Error(\n\nOrigin mismatch))")
+
+        match url {
+            Ok(url) => { assert!(false, "should have error"); }
```

Comment:
A couple small nits:

- Please use `Ok(_)` because otherwise we'll get an unused variable warning. Also, make sure doing a cargo clean + cargo test doesnt emit any other warnings here.
- use `panic!` (which still takes a message) instead of `assert!(false, ...)`. (here and below)